### ISSUE TITLE
Testgrid no longer refreshing Windows 2022 page on `sig-windows-containerd-runtime-signal`.

### ISSUE THREAD

**Issue by aznashwan** [GoogleCloudPlatform/testgrid#779]
TL;DR: the `latest-build.txt` directory whose empty results column can be seen [here](https://testgrid.k8s.io/sig-windows-containerd-runtime-signal#win-2022-containerd-master-integration) was created on accident, was subsequently deleted shortly after (<24h), but testgrid has failed to refresh the page  or show any results past that point (should show at least one result set uploaded per day as can be seen for [sig-windows-containerd-runtime-signal/win-2019-containerd-master-integration](https://testgrid.k8s.io/sig-windows-containerd-runtime-signal#win-2019-containerd-master-integration)).

**Is there any way to force testgrid to refresh/clear caches for `sig-windows-containerd-runtime-signal`?**

Full context:

We're currently facing an issue on the [sig-windows-containerd-runtime-signal/win-2022-containerd-master-integration](https://testgrid.k8s.io/sig-windows-containerd-runtime-signal#win-2022-containerd-master-integration) testgrid page where it appears it got "stuck" on an accidentally created directory.

The directory was created on accident as part of my testing of [this PR](https://github.com/containerd/containerd/pull/6397) which aimed to switch to the new `google-github-actions/upload-cloud-storage` GitHub action library.

One of the testing attempts had misused one of `upload-cloud-storage`'s params which lead to `latest-build.txt` being uploaded under a directory of the same name (i.e. `<Bucket+prefix>/latest-build.txt/latest-build.txt`) instead of the expected `<Bucket+prefix>/latest-build.txt`.

The directory was deleted shortly afterwards (<24h) and, to the best of my knowledge, the directory/file structure is now 100% in line with what it used to be before the mishap.

Is there any way to force testgrid to rescan/re-index that particular bucket?

For reference, the bucket mappings are as follows:
```
#win-2019-containerd-master-integration = gs://contained-integration/logs/windows-ltsc2019 (works fine)
#win-2022-containerd-master-integration = gs://contained-integration/logs/windows-ltsc2022 (is hung up on the 4th of January)

# URI of offending file which caused the hang (had been deleted shortly after its existence was noted)
gs://contained-integration/logs/windows-ltsc2022/latest-build.txt/latest-build.txt

# Correct URI the above should have gone to:
gs://contained-integration/logs/windows-ltsc2022/latest-build.txt
```

Further links:
* [offending action](https://github.com/containerd/containerd/pull/6397/files#diff-ee17ed9b940f6c063117a2fdba0b30b817f3f3ee3d57233be7b924befe715b05R210-R216) (it initially had `destination: ${{ matrix.GOOGLE_BUCKET }}/latest-build.txt` set, which caused the directory to be created by accident, but should be good now)
* [the exact job run which caused this](https://github.com/aznashwan/containerd/runs/4702855807?check_suite_focus=true) (note that the file got misuploaded for 2019 as well but I am unsure why the issue hasn't manifested for the 2019 page too, I can confirm the 2019/latest-build.txt directory had also been deleted shortly after the run)
* [the daily workflow which produces the new artifacts](https://github.com/containerd/containerd/actions/workflows/windows-periodic-trigger.yml) (we should see one result set per Windows version per day (excluding when the job fails), and **I can assure you that the successful jobs' artefacts are present in the bucket and thus should show up on testgrid**)

**Comment by michelle192837** [comment]
The only ways to regenerate state for something like this is:
- for one of us to delete the state and let TestGrid regenerate it
- to create a new test group (e.g. rename the test group); for annotated Prow jobs I think this would mean renaming your job

The former's easier so long as we're aware of it. ^^ I've gone ahead and deleted the state, and TestGrid should update it later today (within several hours or when the job next runs, whichever's first).

**Comment by aznashwan** [comment]
Thanks a lot for the help, seems to have done the trick already!
